packages/styling/src/interfaces/ITypography.ts

```diff
@@ -0,0 +1,60 @@
+export interface IFontFamilies {
+  default: string;
+  monospace: string;
+  [key: string]: string;
+}
+
+export interface IFontSizes {
+  tiny: string;
+  xSmall: string;
+  small: string;
+  medium: string;
+  large: string;
+  xLarge: string;
+  xxLarge: string;
+  xxxLarge: string;
+  mega: string;
+  [key: string]: string;
+}
+
+export interface IFontWeights {
+  light: number | string;
+  default: number | string;
+  bold: number | string;
+  [key: string]: number | string;
+}
+
+export interface IFontColors {
```

Comment:
these colors should be pulled from ISemanticColors

Reply:
Yes agree; we should remove the IFontColors.